mods/species/lizard/datum/culture.dm

```diff
@@ -0,0 +1,8 @@
+/decl/cultural_info/culture/lizard
+	name = CULTURE_LIZARD
+	description = "One of many lizards from known part of space."
```

Comment:
This probably needs a longer description than just 'a space lizard' to be honest.